### PR TITLE
Bids 2857/manage notifications modal

### PR DESCRIPTION
**Pull request by D13ce**
This PR
* Adds a draft for the "Manage Notifications" modal and opens it when the user clicks the "Manage Notifications" button
* Fixes the "Dashboard" icon used on `/notifications`

### CODE REVIEW

File: frontend/components/dashboard/notifications/NotificationsManagementModal.vue [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        </template>
+        General coming soon!
+      </TabPanel>
+      <TabPanel>
+        <template #header>
+          <BcTabHeader :header="$t('dashboard.notifications.management.tabs.dashboards')" :icon="faGaugeSimpleMax" />
+        </template>
+        Dashboards coming soon!
+      </TabPanel>
+      <TabPanel>
```
Comment (MauserBitfly) [inline frontend/components/dashboard/notifications/NotificationsManagementModal.vue:34]: Maybe disable the tabs that are very likely not come in the near future with the development flag

**Review by MauserBitfly — COMMENTED** [review]
Nice job, just a small comment

File: frontend/components/dashboard/notifications/NotificationsManagementModal.vue (hunk 1 above)
Comment (D13ce) [reply frontend/components/dashboard/notifications/NotificationsManagementModal.vue]: Done.

**Review by MauserBitfly — APPROVED** [review]
LGTM

**Pull request merged**